packages/integration-tests/index.js

```diff
@@ -91,9 +91,15 @@ async function waitForAsyncOperationStatus({
  * @returns {string|undefined} - the cluster ARN or undefined if not found
  */
 async function getClusterArn(stackName) {
-  const clusterPrefix = `${stackName}-CumulusECSCluster-`;
-  const listClustersResponse = await ecs().listClusters().promise();
-  return listClustersResponse.clusterArns.find((arn) => arn.includes(clusterPrefix));
+  const { clusterArns } = await ecs().listClusters().promise();
+
+  const matchingArns = clusterArns.filter((arn) => arn.includes(`${stackName}-CumulusECSCluster`));
+
+  if (matchingArns.length !== 1) {
```

Comment:
What was the error case that prompted this?

Reply:
When deploying with Terraform, the resource name is `${stackName}-CumulusECSCluster`. When deploying with kes, the resource name is `${stackName}-CumulusECSCluster-RANDOMSTRING`. It's possible that, if you've deployed with both kes and tf, you can get the wrong result back from that query. If either the kes stack or the tf stack is destroyed, you should only get one result back, which will be the typical case.